README.md

```diff
@@ -106,6 +107,27 @@ Once you have database initialized, you might want to import SSG policies:
 bundle exec rake ssg:import_rhel_supported
 ```
 
+### Project Cyndi
```

Comment:
Can you please add one more heading level, as after this is the continuation of an Option. Or, move it a bit down but copy the init part above.

Text looks good (with internal hyperlinking). Thanks. :-)

Reply:
Yeah :/ I can remove the link if you think we should. I'll fix the header, too.

Reply:
Link removed, header level increased one. Thanks for the review!